src/Components/SightingsPreviewList.js

```diff
@@ -0,0 +1,79 @@
+import axios from 'axios';
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+
+import SightingPreview from './SightingPreview';
+import { BACKEND_URL } from '../constants.js';
+import { Divider, Paper, TextField } from '@mui/material';
+import { Button } from 'react-bootstrap';
+
+const SightingPreviewList = () => {
+  const [sightings, setSightings] = useState([]);
+  const [inputValue, setInputValue] = useState('');
+
+  useEffect(() => {
+    if (inputValue === '')
+      axios.get(`${BACKEND_URL}/sightings`).then((response) => {
+        setSightings(response.data);
+        console.log(response.data[0].MONTH);
+      });
+  }, [inputValue]);
+
+  const handleInputChange = (e) => {
+    setInputValue(e.target.value);
+  };
+
+  const handleButtonClick = () => {
```

Comment:
Since this function is not very generic, how about giving it a more specific name? `handleFilterSightingsByYear` or something along those lines?